apps/files/lib/Command/Scan.php

```diff
@@ -55,48 +55,109 @@ protected function configure(): void {
 
 		$this
 			->setName('files:scan')
-			->setDescription('rescan filesystem')
+			->setDescription('Scans the filesystem for new files and updates the file cache')
```

Comment:
Maybe more explicit, as it does not only detect new files.

```suggestion
			->setDescription('Scans the filesystem and updates the file cache acording to the detected additions, deletions and updates.')
```